two-factor.php

```diff
@@ -52,67 +27,88 @@ function wpcom_vip_force_two_factor() {
 		return false;
 	}
 
-	return apply_filters( 'wpcom_vip_force_two_factor', false );
+	return apply_filters( 'wpcom_vip_is_two_factor_forced', false );
```

Comment:
Should we use the constant here so it auto-flips on the day? Or maybe better to have it intentionally switched over?

```suggestion
	return apply_filters( 'wpcom_vip_is_two_factor_forced', VIP_IS_AFTER_2FA_TIME_GATE );
```

Reply:
That would enable it for everyone. In `wpcom_vip_enforce_two_factor_plugin()`, we calculate `current_user_can( 'edit_posts' )` and then force two factor if that's true.